src/Parser.php

```diff
@@ -27,12 +27,14 @@ class Parser
     private array $endSep;
 
     /**
-     * @param string|string[] $startSep
-     * @param string|string[] $endSep
+     * @param YAMLParser $yamlParser (optional)
+     * @param MarkdownParser $markdownParser (optional)
+     * @param string|string[] $startSep (optional)
+     * @param string|string[] $endSep (optional)
```

Comment:
Can you revert the changes in the phpdoc? They are incorrect and don't bring value on top of the information that the types already provide.